deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -89,13 +83,21 @@ cdk ls --c contextId=demo-deploy
 
 
 Depending on your use-case, you may choose to provide options from both the `cdk.context.json` and the CDK CLI, in which case it is important to know the precedence level for context values. The below order shows these levels with values being passed by the CDK CLI having the most importance
-1. CDK CLI passed context values (highest precedence)
+1. CDK CLI passed context values, e.g. --c stage=dev2 (highest precedence)
 2. Created `cdk.context.json` in the same directory as this README
 3. Existing `default-values.json` in the same directory as this README
 
-## Executing Commands on a Deployed Service
+## Which CDK context options should I use?
+See the [wiki - Configuration Options](https://github.com/opensearch-project/opensearch-migrations/wiki/Configuration-Options) for guidance on which context options should be used for different migrations.
```

Comment:
This seems like its out of order, at the top of `Deploying the CDK` there should be a section about configuring the deployment for your environment that links outward.

I'd also call these deployment configuration options (opposed to cdk context), and you could provide a small hint primmer for what users might want to look for, e.g. Update the file named cdk.context.json for migration options for metadata, historical backfill or traffic capture and replay, see details here [link].

However, as I write this out, it seems like that wiki's content might as well be duplicated here.  What do you think?

Reply:
Thanks have adjusted based on these comments. Personally I'd like to avoid duplicating and just link to the wiki